src/main/java/com/kadai10/user/controller/request/UserUpdateRequest.java

```diff
@@ -1,52 +1,54 @@
 package com.kadai10.user.controller.request;
 
 import lombok.Getter;
+import lombok.Setter;
 
 /**
  * ユーザー情報の更新リクエストを表すクラスです. このクラスはユーザーの名前や職業などの更新が必要な情報を受け取ります。 ユーザー情報を更新する際に使用されます。
  */
-
+@Setter
+@Getter
 public class UserUpdateRequest {
 
   /**
-   * 静的な名前を表すクラス変数. この変数はすべてのインスタンスで共有され、クラスから直接アクセス可能 -- GETTER -- ユーザーの名前を取得するメソッド.
+   * 静的な名前を表すクラス変数. この変数はすべてのインスタンスで共有され、クラスから直接アクセス可能 -- GETTER -- ユーザーの名前を取得するメソッド. -- SETTER
+   * --ユーザーの名前を設定するメソッド.
    */
   @Getter
   private static String name;
```

Comment:
これ前のレビューで気づいていなかったですね。
同時並列的にリクエストがくるWebアプリケーションではstaticなフィールド変数を使うと意図せず他人のリクエスト起因で自分のリクエスト内容が書き換えられることがあります。


ちょっと難しいですが、 static スレッドセーフ などで調べると良いです。
このようにインスタンス変数にしましょう。
```suggestion
  private String name;
```

Reply:
https://github.com/tomoya0844/kadai10/pull/32/commits/5160566df17f6717f32e47b3bf630ad3e855d5b8 このように修正してみました！

Reply:
良いかと！